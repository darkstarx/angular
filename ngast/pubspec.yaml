name: ngast
description: Parser and utilities for AngularDart templates
version: 3.0.0-dev.1
repository: https://github.com/angulardart-community/angular

environment:
  sdk: ">=3.1.0 <4.0.0"

dependencies:
  charcode: ^1.3.1
  collection: ^1.18.0
  meta: ^1.11.0
  source_span: ^1.10.0
  string_scanner: ^1.2.0

dev_dependencies:
  lints: ^3.0.0
  path: ^1.8.3
  test: ^1.24.9
