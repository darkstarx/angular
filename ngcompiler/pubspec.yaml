name: ngcompiler
description: >
  The AngularDart compiler. It compiles your component files written in Dart
  into templates so that you don't have to worry about all the complicated
  hassle.
version: 3.0.0-dev.2
repository: https://github.com/angulardart-community/angular

publish_to: 'none'

environment:
  sdk: ">=3.1.0 <4.0.0"

dependencies:
  analyzer: ^6.3.0
  args: ^2.4.2
  build: ^2.4.1
  code_builder: ^4.8.0
  collection: ^1.18.0
  csslib: ^1.0.0
  dart_style: ^2.3.4
  logging: ^1.2.0
  meta: ^1.11.0
  ngast:
    git:
      url: git@github.com:darkstarx/angular.git
      path: ngast
      ref: dev
  ngdart:
    git:
      url: git@github.com:darkstarx/angular.git
      path: ngdart
      ref: dev
  path: ^1.8.3
  source_gen: ^1.4.0
  source_span: ^1.10.0

dev_dependencies:
  build_test: ^2.2.1
  lints: ^3.0.0
  package_config: ^2.1.0
  test: ^1.24.9
