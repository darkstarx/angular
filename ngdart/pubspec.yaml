name: ngdart
description: >-
  Fast and productive web framework by Google. Separate, but similar to the
  JavaScript framework with a similar name.
version: 8.0.0-dev.3
repository: https://github.com/angulardart-community/angular

publish_to: 'none'

environment:
  sdk: ">=3.1.0 <4.0.0"

dependencies:
  build: ^2.4.1
  built_collection: ^5.1.1
  built_value: ^8.8.0
  collection: ^1.18.0
  intl: ^0.18.1
  js: ^0.6.7
  meta: ^1.11.0
  ngcompiler:
    git:
      url: git@github.com:darkstarx/angular.git
      path: ngcompiler
      ref: dev
  stream_transform: ^2.1.0

dev_dependencies:
  build_runner: ^2.4.7
  build_web_compilers: ^4.0.7
  lints: ^3.0.0
  watcher: ^1.1.0
