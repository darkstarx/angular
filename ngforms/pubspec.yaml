name: ngforms
description: >
  Forms framework for AngularDart. Easily create forms to accept input from
  users.
version: 5.0.0-dev.2
repository: https://github.com/angulardart-community/angular

publish_to: 'none'

environment:
  sdk: ">=3.1.0 <4.0.0"

dependencies:
  meta: ^1.11.0
  ngdart:
    git:
      url: git@github.com:darkstarx/angular.git
      path: ngdart
      ref: dev

dev_dependencies:
  build_runner: ^2.4.7
  build_test: ^2.2.1
  build_web_compilers: ^4.0.7
  lints: ^3.0.0
  mockito: ^5.4.3
  ngtest:
    git:
      url: git@github.com:darkstarx/angular.git
      path: ngtest
      ref: dev
  test: ^1.24.9
