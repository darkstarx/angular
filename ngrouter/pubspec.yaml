name: ngrouter
description: >
  Router for AngularDart. It enables navigation from one view to the next as
  users perform tasks.
version: 4.0.0-dev.2
repository: https://github.com/angulardart-community/angular

publish_to: 'none'

environment:
  sdk: ">=3.1.0 <4.0.0"

dependencies:
  collection: ^1.18.0
  ngdart:
    git:
      url: git@github.com:darkstarx/angular.git
      path: ngdart
      ref: dev

dev_dependencies:
  async: ^2.11.0
  build_runner: ^2.4.7
  build_test: ^2.2.1
  build_web_compilers: ^4.0.7
  lints: ^3.0.0
  mockito: ^5.4.3
  ngtest:
    git:
      url: git@github.com:darkstarx/angular.git
      path: ngtest
      ref: dev
  test: ^1.24.9
