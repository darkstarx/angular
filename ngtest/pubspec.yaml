name: ngtest
description: >
  Testing runner and library for AngularDart. This package is necessary to write
  component tests for AngularDart components.
version: 5.0.0-dev.2
repository: https://github.com/angulardart-community/angular

publish_to: 'none'

environment:
  sdk: ">=3.1.0 <4.0.0"

dependencies:
  collection: ^1.18.0
  meta: ^1.11.0
  ngdart:
    git:
      url: git@github.com:darkstarx/angular.git
      path: ngdart
      ref: dev

dev_dependencies:
  build_runner: ^2.4.7
  build_test: ^2.2.1
  build_web_compilers: ^4.0.7
  lints: ^3.0.0
  test: ^1.24.9
