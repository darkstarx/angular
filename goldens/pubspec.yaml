name: goldens
description: Goldens for AngularDart.
publish_to: none

environment:
  sdk: ">=3.1.0 <4.0.0"

dependencies:
  js: ^0.6.7
  meta: ^1.11.0
  ngdart:
    git:
      url: git@github.com:darkstarx/angular.git
      path: ngdart
      ref: dev

dev_dependencies:
  build_runner: ^2.4.7
  build_web_compilers: ^4.0.7
  dart_style: ^2.3.4
  glob: ^2.1.2
  lints: ^3.0.0
  path: ^1.8.3
